dcmdata/libsrc/dcrleccd.cc

```diff
@@ -465,11 +466,12 @@ OFCondition DcmRLECodecDecoder::decodeFrame(
 
     DcmPixelItem *pixItem = NULL;
     Uint8 * rleData = NULL;
-    const size_t bytesPerStripe = imageColumns * imageRows;
+    const size_t bytesPerStripe = static_cast<Uint32>(imageColumns) * static_cast<Uint32>(imageRows);
     Uint32 numberOfStripes = 0;
     Uint32 fragmentLength = 0;
     Uint32 i;
-    Uint32 frameSize = imageBytesAllocated * imageRows * imageColumns * imageSamplesPerPixel;
+    Uint32 frameSize = static_cast<Uint32>(imageBytesAllocated`) * static_cast<Uint32>(imageRows) 
```

Comment:
I think that frameSize either needs to be size_t  and the static_cast changed to size_t OR  this code needs to be reverted.

Reply:
NOTE:  (imageBytesAllocated`) <-- trailing apostrophe needs to be removed.